database/index.js

```diff
@@ -45,13 +51,31 @@ let save = (reviews) => {
   return Promise.all(savePromises);
 }
 
-// fetch 25 things
-let fetch = () => {
-  return Review.find().sort('_id').limit(25);
+
+let fetchReviews = (callback) => {
+  console.log('fetchReviews invoked! Serving you 10 reviews 😀');
+  Review.find(null, null, {
+    limit: 10
+
+  }, (error, docs) => {
+    if(error) {
+      callback(error)
+    } else {
+      callback(null, docs);
+    }
+  })
 }
+
+
+let fetchByProductId = (productID) => {
+  console.log('fetchByProductId invoked! Param is ', productID);
+  return Review.findOne({product_id: productID});
```

Comment:
The codes look clean.
I like to suggest for you to using ES6 style in the future.
You change the productID parameter to product_id. Then, you can simply use ({product_id}) instead of ({product_id: productID}) as a findOne argument.